deploy/gcloud/README.md

```diff
@@ -118,6 +118,60 @@ The following information can be customized in setup.sh, but the default also wo
 
   kubectl apply -f amppackager_service.yaml
 
+## Production Reverse Proxy Setup
+
+For now, productionizing is a bit manual. The minimum steps are:
+
+  1. Don't expose `amppkg` to the outside world; keep it on your internal
```

Comment:
Change to:

> Don't expose `amppkg` to the outside world. If possible, keep it on your internal network. In addition, use the loadBalancerSourceRanges above so that it only responds to the reverse proxy you set up.

Since these are separate defenses.

[also nit: setup (n) vs set up (v)]

Reply:
Done.